website_sale_wishlist/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
I guess you should add simple headers to all the .py files, check the whole module template https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py